docsrc/hello_world.rst

```diff
@@ -82,44 +82,65 @@ which contains all messages written to the console.
 If the ``output_dir`` argument is omitted, the output files are written
 to a temporary directory which is deleted when the current Python session is terminated.
 
+.. code-block:: python
 
-Access the sample
-^^^^^^^^^^^^^^^^^
+    # specify data file
+    bernoulli_data = os.path.join(cmdstan_path(), 'examples', 'bernoulli', 'bernoulli.data.json')
 
-The :ref:`class_cmdstanmcmc` object stores the CmdStan config information and
-the names of the the per-chain output files.
-It manages and retrieves the sampler outputs as Python objects.
+    # fit the model 
+    bern_fit = bernoulli_model.sample(data=bernoulli_data, output_dir='.') 
 
-.. code-block:: python
+    # printing the object reports sampler commands, output files
+    print(bern_fit)
 
-   print(bern_fit)
 
-The resulting set of draws produced by the sampler is lazily instantiated
-as a 3-D ``numpy.ndarray`` (i.e., a multi-dimensional array)
-over all draws from all chains  arranged as draws X chains X columns.
-Instantiation happens the first time that any of the information
-in the posterior is accesed via properties:
-``draws``, ``metric``, or ``stepsize`` are accessed.
-At this point the stan-csv output files are read into memory.
-For large files this may take several seconds; for the example
-dataset, this should take less than a second.
+Access the sample
+^^^^^^^^^^^^^^^^^
+The CmdStanMCMC class has properties and methods to access the sampler
+outputs and all metadata. Accessor properties include:
+
+ * draws - all draws from all chains, stored as a 3D numpy.ndarray.
```

Comment:
Summary, stan_variable, and stan_variables should go here. I think `summary` and `stan_variable` would be the two things I'd use the most with cmdstanpy.